server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/actor/Actor.java

```diff
@@ -0,0 +1,92 @@
+package de.tum.in.www1.hephaestus.codereview.actor;
+
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.hephaestus.codereview.comment.Comment;
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "actors")
+@Getter
+@Setter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Actor {
+    /**
+     * Unique identifier for a User entity.
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    /**
+     * Login of the User entity.
+     * This field is mandatory.
+     */
+    @Column(nullable = false)
+    private String login;
+
+    @Column
+    private String email;
+
+    /**
+     * URL of the User entity.
+     * This field is mandatory.
+     */
+    @Column(nullable = false)
+    private String url;
+
+    /**
+     * The Pullrequests of the User entity.
+     */
+    @OneToMany(mappedBy = "author")
+    @JsonIgnore
+    private List<Pullrequest> pullrequests;
+
+    /**
+     * The Comments of the User entity.
```

Comment:
I meant that maybe some comments are obvious from the from the code and don't need further explanation that repeats it.

For instance:
```
/**
 * Unique identifier for a Comment entity.
 */
@Id
@GeneratedValue(strategy = GenerationType.IDENTITY)
private Long id;
```

I know that this is a field of the Comment entity, it's tagged with `@Id` so I know it is a identifier and also unique.

Another example:
```
/**
 * Body of the Comment entity.
 * This field is mandatory.
 */
@NonNull
private String body;
```

I think in such cases we can omit the comment because it does not say more than the code itself. What do you think?